code/modules/modular_computers/file_system/picture_file.dm

```diff
@@ -0,0 +1,39 @@
+/**
+ * PNG file type
+ * Stores a picture to be used for programs.
+ */
+
+GLOBAL_VAR_INIT(next_ntos_picture_uid, 0)
```

Comment:
This could/should be a static instead.

Reply:
How can I rewrite this into a static var? I'm not super familiar with the syntax